### DISCUSSION TITLE
Some problems with Nvidia video cards

### DISCUSSION THREAD

**Discussion by Bravleubiytsabotov2013** [JaKooLit/Fedora-Hyprland#97] [Q&A]
I know that hyprland does not work well with Nvidia video cards, but even with all the fixes that they offer in their guides?
I tried to solve all the problems that arose for me, but it didn’t work out for me.
I encountered graphical errors, as well as some interface not being smooth, as if it were at 20 fps - this is normal for hyprland?

**Comment by JaKooLit** [comment]
no its not normal. Go through readme.. 

plus all depends which driver you installed

**Reply by Bravleubiytsabotov2013** [reply]
Hi, I followed the instructions from the readme.  Approximately the same problems that I described are present not only in hyprland, but also in gnome and kda.  I use the Nobara 39 distribution, and the driver is proprietary from Nvidia.  I also edited the "ENVariables.conf" file, but in truth I only enabled or enabled the values ​​and rebooted and so on from the beginning to the end of this file.
